types/react-native-touch-id/react-native-touch-id-tests.ts

```diff
@@ -0,0 +1,5 @@
+import { isSupported, authenticate } from 'react-native-touch-id';
+
+isSupported().then(() => {}).catch(() => {});
+
+authenticate('reason').then(() => {}).catch(() => {});
```

Comment:
Something like that maybe yeah, as long as it would fail when the wrong types are used by the user.